openstack/autoscaling/v1/plannedtasks/results.go

```diff
@@ -0,0 +1,75 @@
+package plannedtasks
+
+import (
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// CreateResp is the structure that represents the response to creation planned task.
+type CreateResp struct {
```

Comment:
```suggestion
type createResp struct {
```
only current package used

Reply:
fixed.